### ISSUE TITLE
[feat] Remove use of jQuery

### ISSUE THREAD

**Issue by cptcrunchy** [xaviersalazar/check-exercises-app#12]
Could we systematically remove reference to the jQuery library? not much else to say really.

**Comment by xaviersalazar** [comment]
You mean like remove jQuery all together and be replaced with vanilla JS?

**Comment by cptcrunchy** [comment]
yes. In my humble opinion, jQuery isn't needed unless there is some library that will not function without including jQuery.

**Comment by xaviersalazar** [comment]
That makes sense to me. If you wanna give it a go you sure can. Just remove the scripts and the dependency from the package
